packages/markdown/index.ts

```diff
@@ -22,13 +31,13 @@ function assert(condition: unknown, message: string): asserts condition {
 	}
 }
 
-export function create(): ServicePlugin {
+export function create(options: CreateOptions): ServicePlugin {
```

Comment:
Let's make this option optional to be consistent with other services.

Reply:
Then we’d need a default value. I don’t really think there’s a good default.

Reply:
I just found out that options are for passing "markdown.validate". What Editor needs to define it? Can it be hardcoded as "markdown.validate" in the service?

Reply:
IMO it makes sense to use `mdx.validate` for example.